cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,19 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+REGISTRY?="gcr.io"
+PROJECT?="google_containers"
+
 all:
-	docker build --pull -t gcr.io/$(PROJECT)/kubemark .
-	gcloud docker -- push gcr.io/$(PROJECT)/kubemark
+	gcloudpush
+
+build:
+	docker build --pull -t $(REGISTRY)/$(PROJECT)/kubemark .
+
+gcloudpush: build
```

Comment:
Sounds good. I'm fine with either letting it stay the same as now or making 'docker' the default instead of 'gcloud docker' (with a slight preference for this).
Other than that it's an lgtm, but I'll let @roberthbailey give the final lgtm.